src/BSON/Binary.c

```diff
@@ -142,6 +146,21 @@ static void php_phongo_binary_free_object(void *object TSRMLS_DC) /* {{{ */
 	efree(intern);
 } /* }}} */
 
+#if PHP_VERSION_ID >= 70000
+zend_object* php_phongo_binary_create_object(zend_class_entry *class_type TSRMLS_DC) /* {{{ */
+{
+        php_phongo_binary_t *intern;
+
+        intern = (php_phongo_binary_t *)ecalloc(1, sizeof(php_phongo_binary_t)+zend_object_properties_size(class_type));
+
+        zend_object_std_init(&intern->std, class_type TSRMLS_CC);
+        object_properties_init(&intern->std, class_type);
+
+        intern->std.handlers = &php_phongo_handler_binary;
+
+        return &intern->std;
+}
```

Comment:
The closing folding markers, /* }}} */, are missing (looks like they are missing